Microsoft.Toolkit.Uwp.Notifications/Microsoft.Toolkit.Uwp.Notifications.nuspec

```diff
@@ -6,9 +6,18 @@
     <title>Windows Community Toolkit Notifications</title>
     <authors>Microsoft.Toolkit,dotnetfoundation</authors>
     <requireLicenseAcceptance>true</requireLicenseAcceptance>
-    <licenseUrl>https://github.com/windows-toolkit/WindowsCommunityToolkit/blob/master/license.md</licenseUrl>
-    <projectUrl>https://github.com/windows-toolkit/WindowsCommunityToolkit</projectUrl>
-    <iconUrl>https://raw.githubusercontent.com/windows-toolkit/WindowsCommunityToolkit/master/build/nuget.png</iconUrl>
+  <projectUrl>https://github.com/windows-toolkit/WindowsCommunityToolkit</projectUrl>
+<package>
+  <metadata>
+    <icon>images\nuget.png</icon>
+    <iconUrl>images\nuget.png</iconUrl>
+    <license type="file">License.md</license>
+  </metadata>
+  <files>
+    <file src="..\build\nuget.png" target="images\nuget.png" />
+    <file src="..\license.md" target="" />
+  </files>
+</package>
```

Comment:
```suggestion
    <projectUrl>https://github.com/windows-toolkit/WindowsCommunityToolkit</projectUrl>  
    <iconUrl>https://raw.githubusercontent.com/windows-toolkit/WindowsCommunityToolkit/master/build/nuget.png</iconUrl>
    <icon>images\nuget.png</icon>
    <license type="file">License.md</license>
```

We're already part of the metadata section here.

Then just move these two lines into the existing files section below:
```
    <file src="..\build\nuget.png" target="images\nuget.png" />
    <file src="..\license.md" target="" />
```